docs/workspace-sync.html.md

```diff
@@ -57,20 +57,20 @@ authentication.
    `https://app.terraform.io/app/$ORGANIZATION/settings/teams`, where
    `$ORGANIZATION` is your organization name.
 
-1. Create a file named `credentials` in a memorable directory.
+1. Create a file for storing the API token and open it in a text editor.
 
 1. Insert the generated token (`$TERRAFORM_CLOUD_API_TOKEN`) into the
-   `credentials` formatted for Terraform credentials.
+   text file formatted for Terraform credentials.
    ```hcl
    credentials app.terraform.io {
      token = "$TERRAFORM_CLOUD_API_TOKEN"
    }
    ```
 
-1. Create a Kubernetes secret named `terraformrc` for the namespace from the
-   `credentials` file.
+1. Create a Kubernetes secret named `terraformrc` in the namespace.
+   Reference the credentials file (`$FILENAME`) created in the previous step.
    ```shell
-   $ kubectl create -n $NAMESPACE secret generic terraformrc --from-file=credentials
+   $ kubectl create -n $NAMESPACE secret generic terraformrc --from-file=credentials=$FILENAME
```

Comment:
I added the key name (`credentials`) here because I got stuck on this, as a new user, when I tried to name my file something other than `credentials`.

Reply:
Got it. So does this mean that the `sshKeyID` needs to be added to the `terraformrc` secret as another `--from-file`? Or otherwise?

I don't know if maybe we can update the `values.yaml` to point out that `terraformRC.secretKey` needs to be equivalent to this.

Reply:
This change is unrelated to the SSH key work. I was just setting up my environment to be able to test the operator.

For the SSH key, I had to paste it into the TFC console. But I like your idea of using something in kubernetes to send that information to TFC instead. We might need to create a new issue to track that enhancement.